Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -130,7 +130,7 @@ def audit
       formula_count += 1
       problem_count += fa.problems.size
       problem_lines = format_problem_lines(fa.problems)
-      corrected_problem_count = options.fetch(:style_offenses, []).count(&:corrected?)
+      corrected_problem_count = options[:style_offenses].count(&:corrected?)
```

Comment:
Isn't this going to have the problem from before again or is it always set now?

Reply:
Always set https://github.com/Homebrew/brew/blob/32bc5535b6e365959bdf88533cdfa5e1271ea6fa/Library/Homebrew/style.rb#L122